blocks/to-do/add-list-items.js

```diff
@@ -1,11 +1,28 @@
 export default function addListItem(event, inputValue, list) {
-  // Prevent button from scrolling up
-  event.preventDefault();
-
   // Make sure input is not empty
   if (inputValue !== '') {
+    // Create elements
+    const listItemDiv = document.createElement('div');
     const listItem = document.createElement('li');
+    const listItemBtn = document.createElement('button');
```

Comment:
Why the extra div? `li` is already a block-level element that can hold the label and the button as well.
Not an issue, just asking why :)